sdk/python/responsible-ai/mlflow-deployment-with-explanations/mlflow-deployment-with-explanations.ipynb

```diff
@@ -0,0 +1,904 @@
+    "categorical_features = []  # Optional, will be calculated from baseline data if None\n",
+    "\n",
+    "# Enter baseline data information (must match Azure ML asset)\n",
+    "baseline_data_name = \"<BASELINE_DATA_NAME>\"  # Name of the baseline data\n",
+    "baseline_data_version = \"<BASELINE_DATA_VERSION>\"  # Version of that data\n",
+    "\n",
+    "# Enter wrapper model name\n",
+    "wrapper_model_name = \"<WRAPPER_MODEL_NAME>\"  # Name for the model with explanations"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_uri = f\"models:/{model_name}/{model_version}\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2.2 Load baseline data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The data used to create the explanation wrapper must match the data used to train the original model. Be sure to drop any columns that were dropped during training. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import mltable\n",
+    "\n",
+    "data_asset = ml_client.data.get(name=baseline_data_name, version=baseline_data_version)\n",
+    "baseline_df = mltable.load(f\"azureml:/{data_asset.id}\").to_pandas_dataframe()\n",
+    "# Drop any columns that were dropped when training the model. Uncomment and fill in first parameter\n",
+    "# baseline_df = baseline_df.drop([], axis=\"columns\")\n",
+    "\n",
+    "print(\"Baseline Data (first 5 rows):\")\n",
+    "baseline_df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2.3 Create instance of ExplanationWrapper"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from explanation_wrapper import ExplanationWrapper\n",
+    "\n",
+    "explanation_wrapper = ExplanationWrapper(\n",
+    "    model_uri=model_uri,\n",
+    "    baseline_df=baseline_df,\n",
+    "    target_column=target_column,\n",
+    "    task_type=task_type,\n",
+    "    categorical_features=categorical_features,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Register Wrapper Model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3.1 Download original model **(user input required)**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Download the original model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ml_client.models.download(\n",
+    "    name=model_name, version=model_version, download_path=\"./artifacts/original_model\"\n",
+    ")"
+   ]
+  },
```

Comment:
Since this tutorial is already using Mlflow, I suggest to use the MLflow syntax to download the assets needed. In that way, the user follows 1 single SDK.